### ISSUE TITLE
secondIssue_CRUD구현

### ISSUE THREAD

**Issue by potterLim** [s21900772/miniProject#3]
함수들 구현에 관해 논의해야할것 같습니

**Comment by choihjin** [comment]
네 그럼 4개 함수를 구현해야하니까 2개씩 나눠서 하는건 어떨까요??

**Comment by potterLim** [comment]
혹시 하시고싶은 함수 있으시면 남는거 제가 해도 괜찮은데 혹시 하시고싶은거 있으신가요?

**Comment by choihjin** [comment]
저도 큰 상관은 없는데, 그럼 제가 create read함수 하겠습니다

**Comment by potterLim** [comment]
예 그럼 제가 updatae delete함수 하겠습니다. delete된 정보는 date를 "-1"로 초기화 하겠습니다. read할때 참고해서 read해주세요!

**Comment by choihjin** [comment]
네 -1로 반영하겠습니다!

**Comment by choihjin** [comment]
selectMenu함수 void에서 int타입으로 변경했고 selectMenu, create, read함수 구현했습니다, 메인함수랑 수정, 삭제 구현해주시면 될거같습니다!

**Comment by potterLim** [comment]
옙 감사합니다 확인 이후에 구현하도록 하겠습니다!

**Comment by potterLim** [comment]
우선 main함수 구현하였고 함수원형을 약간 수정하였습니다 이후에 delete, update함수 구현해서 push하도록 하겠습니다

**Comment by potterLim** [comment]
delete, update 함수 구현 완료해서 push했습니다.

**Comment by choihjin** [comment]
네 확인했습니다. merge 하여서 최종 테스트 해보면 될거같습니다!

**Comment by choihjin** [comment]
가벼운 테스트케이스는 문제없이 돌아가네요!

**Comment by potterLim** [comment]
Screenshots 업로드해서 push했습니다 확인 부탁 드립니다

**Comment by potterLim** [comment]
사진을 제외한 함수구현에 대한 wiki 작성 했습니다.

**Comment by choihjin** [comment]
총 금액에 대한 출력과 변수 선언 수정했습니다!

**Comment by choihjin** [comment]
스크린샷까지 수정완료 하였습니다!!

**Comment by potterLim** [comment]
새로운 md파일 PR했습니다 확인하고 피드백 주시면 곧장 수정하도록 하겠습니다
